uxarray/helpers.py

```diff
@@ -673,3 +673,49 @@ def close_face_nodes(Mesh2_face_nodes, nMesh2_face, nMaxMesh2_face_nodes):
     np.put(closed.ravel(), first_fv_idx_1d, first_node_value)
 
     return closed
+
+
+def _convert_face_node_conn_to_sparse_matrix(dense_matrix: np.ndarray) -> tuple:
+    """Converts a given dense matrix connectivity to a sparse matrix format
+    where the locations of non fill-value entries are stored using COO
+    (coordinate list) standard. It is represented by three arrays: row indices,
+    column indices, and non-zero element flags.
+
+    Parameters
+    ----------
+    dense_matrix : np.ndarray
+        The dense matrix to be converted.
+
+    Returns
+    -------
+    tuple
+        A tuple containing three arrays:
+
+        - face_indices : np.ndarray
+            Array containing the face indices for each non-zero element.
+        - node_indices : np.ndarray
+            Array containing the node indices for each non-zero element.
+        - non_zero_elements_flag : np.ndarray
+            Array containing flags indicating if a non-zero element is present in the corresponding row and column
+            index.
```

Comment:
Change the description and variable name to mention non-fill-value entries instead of non-zero